components/spider-proto/common/common.proto

```
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package common;
```

Buf (1.70.0)

[error] Files with package "common" must be within a directory "common" relative to root but were in directory "components/spider-proto/common".

(PACKAGE_DIRECTORY_MATCH)